openapi/servicenow/client.bal

```diff
@@ -0,0 +1,477 @@
+        string  path = string `/api/now/table/${tableName}`;
+        map<anydata> queryParam = {"name-value pairs": nameValuePairs, "sysparm_display_value": sysparmDisplayValue, "sysparm_exclude_reference_link": sysparmExcludeReferenceLink, "sysparm_fields": sysparmFields, "sysparm_limit": sysparmLimit, "sysparm_no_count": sysparmNoCount, "sysparm_offset": sysparmOffset, "sysparm_query": sysparmQuery, "sysparm_query_category": sysparmQueryCategory, "sysparm_query_no_domain": sysparmQueryNoDomain, "sysparm_suppress_pagination_header": sysparmSuppressPaginationHeader, "sysparm_view": sysparmView};
+        path = path + check getPathForQueryParam(queryParam);
+        map<any> headerValues = {"Accept": accept};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        json response = check self.clientEp-> get(path, accHeaders, targetType = json);
+        return response;
+    }
+    # Inserts one record in the specified table. Multiple record insertion is not supported by this method.
+    #
+    # + tableName - Name of the table in which to save the record.
+    # + payload - Field name and the associated value for each parameter to define in the specified record in JSON format.
+    # + sysparmDisplayValue - Data retrieval operation for reference and choice fields.
+    # + sysparmExcludeReferenceLink - Flag that indicates whether to exclude Table API links for reference fields
+    # + sysparmFields - Comma-separated list of fields to return in the response.
+    # + sysparmInputDisplayValue - Flag that indicates whether to set field values using the display value or the actual value.
+    # + sysparmView - UI view for which to render the data. Determines the fields returned in the response.
+    # + accept - Data format of the response body.
+    # + contentType - Data format of the request body.
+    # + xNoResponseBody - By default, responses include body content detailing the new record. Set this header to true in the request to suppress the response body.
+    # + return - If successful, returns a JSON object that includes created record of the table. Otherwise returns the relevant error.
+    remote isolated function insertRecordToTable(string tableName, json payload, string? sysparmDisplayValue = (), boolean? sysparmExcludeReferenceLink = (), string? sysparmFields = (), boolean? sysparmInputDisplayValue = (), string? sysparmView = (), string? accept = (), string? contentType = (), string? xNoResponseBody = ()) returns json|error {
```

Comment:
The function name `insertRecordToTable` sounds odd. Isn't it correct to say `createRecord`? It is obvious that record is adding to a table because we have to give the `tableName`.

Reply:
Fixed by https://github.com/kasthuriraajan/ballerinax-openapi-connectors/commit/7d722e1656dd4084768a6a6f0aab1753b97a5d98